scripts/run_sim_benchmark.py

```
@@ -0,0 +1,116 @@
+    env["CROSSPOINT_SIM_SD"] = str(sd_root)
+    env["SDL_VIDEODRIVER"] = args.video_driver
+    if not args.keep_cache:
+        env["CROSSPOINT_SIM_BENCH_CLEAR_CACHE"] = "1"
+    if args.heap_bytes > 0:
+        env["CROSSPOINT_SIM_HEAP_BYTES"] = str(args.heap_bytes)
+    # Heap caller tracking is on by default so an OOM prints the call sites
+    # holding the arena, not just free/max_alloc totals.
+    env["CROSSPOINT_SIM_HEAP_TRACE"] = "0" if args.no_heap_trace else "1"
+
+    cmd = ["python3", "-m", "platformio", "run", "-e", args.env, "-t", "run_simulator"]
+    return subprocess.run(cmd, cwd=repo_root, env=env, check=False).returncode
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)